gdal/frmts/pds/pds4dataset.cpp

```diff
@@ -4084,6 +4084,8 @@ PDS4Dataset *PDS4Dataset::CreateInternal(const char *pszFilename,
         if( poSrcDriver)
         {
             auto pszDriverName = poSrcDriver->GetDescription();
+            papszOptions = CSLSetNameValue(
+                papszOptions, "VAR_VERTICAL_DISPLAY_DIRECTION", "Top to Bottom");
```

Comment:
It is not clear to me if the variable should only be used when `CREATE_LABEL_ONLY = YES`